packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -563,12 +567,36 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         return tree.entries;
     }
 
+    /**
+     * Returns the Fluid objects referenced by this channel.
+     */
+    private getFluidObjectReferences(): IFluidObjectReferences {
```

Comment:
Based on what this API does and how it's used, I'd call it getGcNodeForDataStore(), or simply getGcNode() (given that's API on data store). It returns single node (with properties describing this node), not references.